src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void print();
```

Comment:
```suggestion
    void draw();
```

Reply:
not explicitly required in task description, but would make sense to name the only method of Drawable - draw